app/src/main/java/pm/gnosis/heimdall/data/repositories/BridgeRepository.kt

```diff
@@ -31,17 +34,17 @@ interface BridgeRepository {
         ) : SessionEvent()
     }
 
-    fun sessions(): Single<List<SessionMeta>>
+    fun sessions(safe: Solidity.Address?): Single<List<SessionMeta>>
```

Comment:
why is safe nullable? Is it possible to have a session that is not attached to any safe?

Reply:
null would mean no filter (so display all available sessions)

Reply:
would it make sence to have 2 methods? One that returns all sessions (Single<List<SessionMeta>>) and one that returns sessionForSafe (Single<SessionMeta>)?